src/services/user/services/SkipRegistration.js

```diff
@@ -0,0 +1,97 @@
+const { BadRequest, Forbidden } = require('@feathersjs/errors');
+
+const validateRequest = (data, targetUser, actingUser) => {
+	// check persmissison
+	const targetIsStudent = targetUser.roles[0].name === 'student';
+	const targetIsTeacher = targetUser.roles[0].name === 'teacher';
+	let hasPermission = false;
+
+	if (targetIsStudent && actingUser.permissions.includes('STUDENT_SKIP_REGISTRATION')) {
+		hasPermission = true;
+	}
+	if (targetIsTeacher && actingUser.permissions.includes('TEACHER_SKIP_REGISTRATION')) {
+		hasPermission = true;
+	}
+	// school
+	if (targetUser.schoolId !== actingUser.schoolId) hasPermission = false;
```

Comment:
There appears to be an issue with this comparison. While the content of the two compared objects appear to be the same, the actual objects are different and that's why a user isn't allowed to change everything. 
The issue can be fixed by adding `toString()` to each object, though that seems to be a dirty solution.